pkg/controller/configmap/configmap_controller.go

```diff
@@ -588,7 +588,7 @@ func (r *ReconcileConfigMap) isSecretChanged(opNsxSecret *corev1.Secret, opLbSec
 		}
 		ncpNsxSecret = nil
 	}
-	nsxSecretChanged := secretEqual(opNsxSecret, ncpNsxSecret)
+	nsxSecretChanged := secretNotEqual(opNsxSecret, ncpNsxSecret)
```

Comment:
I believe func name :secretEqual is not semantically correct according to function purpose.
I suggest it be changed to secretNotEqual although its intrinsic logic is correct.

Reply:
Move this change to PR #116 .
thanks